etest/docker/container.py

```diff
@@ -16,8 +16,7 @@
 
 def commit(container: Container, tag: str, repository: str, *args, **kwargs):
     """Commit a Docker container."""
-    print(repository)
-    repo = repository.replace("=", "").replace("[", "").replace("]", "").replace(",", "")
+    repo = "".join([c for c in repository if c not in "=[],"]).lower()
```

Comment:
Did the cast not work?  It's a bit more direct than using the join if we can.

Reply:
Nope. It returns something like "'t','e','s','t,'"

Reply:
That's too bad.  Well, this will work but we can keep the chained replaces if preferred.

Although going back to the regular expression would work as well:

re.sub(r"[=[\],]", "", repository)